CMakeLists.txt

```diff
@@ -69,13 +69,18 @@ else()
     endif()
 endif()
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${CPP_FLAGS} ${CXX11_FLAG}")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${CPP_FLAGS} ${CXX11_FLAG} -Wall -Wextra")
```

Comment:
There's a small chance that these flags might have to get reverted if I go messing around with windows compilers, but it's not a blocker for merging.